### PR TITLE
[patch] Fixes for MAS WSLCfg generation 

### PR DESCRIPTION
**Pull request by andrercm**
To close the gap on https://github.com/ibm-mas/ansible-devops/issues/192

### CODE REVIEW

**Comment by andrercm** [comment]
Works with cpd40, will run regression to ensure it also works with cpd35

File: ibm/mas_devops/roles/cp4d_install_services/templates/watsonstudiocfg.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    mas.ibm.com/configScope: system
+    mas.ibm.com/instanceId: "{{mas_instance_id}}"
+spec:
+  config:
+    credentials:
+      secretName: "{{ mas_instance_id }}-usersupplied-watsonstudio-creds-system"
+    endpoint: "{{ cpd_url }}"
+  displayName: "MAS - Watson Studio configuration"
+  type: external
+
```
Comment (durera) [inline ibm/mas_devops/roles/cp4d_install_services/templates/watsonstudiocfg.yml.j2:26]: Github reports no newline at end of file ... current linting lets this pass for j2 files, but future won't, so please add missing newline

File: ibm/mas_devops/roles/cp4d_install_services/defaults/main.yml [hunk 2]
```diff
@@ -2,3 +2,8 @@
 cpd_version: "{{ lookup('env', 'CPD_VERSION') }}"
 cpd_storage_class: "{{ lookup('env', 'CPD_STORAGE_CLASS') }}"
 cpd_wd_storage_class: "{{ lookup('env', 'CPD_WD_STORAGE_CLASS') }}"
+cpd_admin_username: admin
+
+# MAS Configuration
+mas_instance_id: "{{ lookup('env', 'MAS_INSTANCE_ID') }}"
```
[the remark this answers is not in the document]
Comment (andrercm) [reply ibm/mas_devops/roles/cp4d_install_services/defaults/main.yml:8]: Done

File: ibm/mas_devops/roles/cp4d_install_services/templates/watsonstudiocfg.yml.j2 (hunk 1 above)
Comment (andrercm) [reply ibm/mas_devops/roles/cp4d_install_services/templates/watsonstudiocfg.yml.j2]: I thought row 26 was enough as extra new line? in any case i added one more... is it ok now?

**Review by lokeshs09 — DISMISSED** [review]

**Review by durera — DISMISSED** [review]

**Review by durera — DISMISSED** [review]

**Review by durera — APPROVED** [review]

**Pull request merged**